src/Components/Artwork/GridItem.tsx

```diff
@@ -61,6 +63,14 @@ export const ArtworkGridItem: React.FC<ArtworkGridItemProps> = ({
     rest.onMouseLeave?.(event)
   }
 
+  const localImagePlaceholder =
```

Comment:
**praise:** nice touch

Reply:
**Q** I don't get what you are trying to do here 👀 would you mind maybe explaining it to me? we can later add a comment to the code here maybe